Makefile

```
@@ -1,4 +1,4 @@
-.PHONY: install serve build clean preview check-images
+.PHONY: install serve build clean preview check-images test
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)